classes/local/store/object_file_system.php

```diff
@@ -450,7 +499,6 @@ public function remove_file($contenthash) {
             // Don't remove the file - it's still in use.
             return;
         }
-
```

Comment:
typo?

Reply:
not sure about this one, I can't find this change locally :/